bin/node_modules/nopt/.npmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This file introduced by nopt, not by me. Anyway we're shipping node dependencies bundled in this package so this won't affect en-user.
@agrieve Your thoughts?